src/bindings/python/tests/test_runtime/test_output_node.py

```diff
@@ -31,3 +31,19 @@ def test_output_names():
     more_names = {"yet_another_name", "input1"}
     param.output(0).add_names(more_names)
     assert param.output(0).get_names() == names.union(more_names)
+
+def test_tensor_bounds():
+    param = ops.parameter([1, 64], Type.f32)
+    tensor = param.output(0).get_tensor()
+    values = Tensor(np.zeros([1, 64], dtype=np.float32))
+    tensor.set_lower_value(values)
+    tensor.set_upper_value(values)
+    assert np.array_equal(tensor.get_lower_value().data, values.data) and np.array_equal(tensor.get_upper_value().data, values.data)
+
+def test_output_symbol():
+    param = ops.parameter([1,64], Type.f32)
```

Comment:
```suggestion
    param = ops.parameter([1, 64], Type.f32)
```
to fix code style job